plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -73,7 +75,13 @@ const Form = withTheme(require('@rjsf/material-ui-v5').Theme);
  * The `Stepper` component is the Wizard that is rendered when a user selects a template
  * @alpha
  */
-export const Stepper = (props: StepperProps) => {
+
+export const Stepper = ({
+  ReviewStateWrapper = ReviewState,
```

Comment:
Let's destructure these things inside the component and assign defaults in there. Otherwise it makes the `api-report` pretty tricky to read. https://github.com/backstage/backstage/pull/15162/files#diff-30803124f64dd6757ee54c8081a2bb05c00dbbdb1f90fa386203a8c5ddceb9e7R351-R356

You can still assign defaults in the destructuring inside the component.